### ISSUE TITLE
Cutting CNI volumes

### ISSUE THREAD

**Issue by leotozzi88** [WilliamsPANLab/Pipelines#2]
Calibration volumes are included in the CNI fMRI and DWI scans. 
These include:

- 2 scans for fMRI
- 3 scans for SBRef
- 3 scans for Fieldmaps
- 2 scans for DWI

These should be cut before the images are processed. At the moment this step is embedded in the conversion script, eventually it should be a script that follows conversion but precedes processing.

**Comment by bstavel** [comment]
I agree. I think there should likely be a Bidsisfication script, followed by a validation script, where we check for the right number of volumes, and cut the dummy volumes
